locale/Makefile

```diff
@@ -31,32 +32,28 @@ all-mo: $(MOFILES)
 check: $(POXFILES)
 	msgfmt -c ${POTFILE}
```

Comment:
Mind fixing this line? It's actually incorrect. `msgfmt` on POTFILE won't work once it is regenerated with the invalid heaer. Just delete the line. We won't check the POTFILE (pofilter does not make sense here - all the translations are empty in POT).